contracts/HubPool.sol

```diff
@@ -405,6 +421,37 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
                 .rawValue;
     }
 
+    function _executeRelayerRefundOnTargetChain(MerkleLib.PoolRebalance memory poolRebalance, uint256 refundRequestId)
+        internal
+    {
+        AdapterInterface adapter = crossChainContracts[poolRebalance.chainId].adapter;
+        adapter.relayMessage(
+            address(adapter),
+            abi.encodeWithSignature("initializeRelayerRefund(address)", refundRequest.destinationDistributionRoot)
```

Comment:
This should be "bytes32" not "address" right?

Reply:
it should! fixed.